### PR TITLE
SEO-173280-Image-Alt-TextBox-WPF

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the links with redirects and changing image names and alt text.

Regards,
Beryl.

### CODE REVIEW

File: wpf/PropertyGrid/Getting-Started.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ propertyGrid1.SetBinding(PropertyGrid.SelectedObjectProperty, new Binding("Selec
 {% endhighlight %}
 {% endtabs %}
 
-![ Populating the SelectedEmployee object properties into the PropertyGrid control](Getting-Started_images/Binding-with-any-object_img1.png)
+![ Populating the SelectedEmployee object properties into the PropertyGrid control](getting-started_images/wpf-propertygrid-populating-properties)
```
Comment (MallikaRK) [inline wpf/PropertyGrid/Getting-Started.md:165]: image file format name is missing. Please update properly.
.png is missing here.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/PropertyGrid/Getting-Started.md (hunk 1 above)
Comment (berylouma) [reply wpf/PropertyGrid/Getting-Started.md]: Updated image name properly

**Review by Girija-Rajendran — APPROVED** [review]

**Review by MallikaRK — APPROVED** [review]
Approved

**Pull request merged**